src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,170 @@ Jon Schneider <jschneider@pivotal.io>
 :sectnums:
 :system: dynatrace
 
-Dynatrace is a dimensional time-series SaaS with built-in dashboarding.
+https://www.dynatrace.com/[*Dynatrace*] is a Software Intelligence Platform featuring application performance monitoring (APM), artificial intelligence for operations (AIOps), IT infrastructure monitoring, digital experience management (DEM), and digital business analytics capabilities.
+It can ingest multi-purpose dimensional time-series data and has built-in dashboarding. Both SaaS and self-hosted (Managed) deployments are offered.
 
 include::install.adoc[]
 
 == Configuring
 
+=== Setup
+
+For setting up new integrations with Dynatrace, it is recommended to use the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[latest version] of the https://www.dynatrace.com/support/help/dynatrace-api/[API] (`api-version: v2`).
+
+Use the following code in your project to export Micrometer metrics to the specified Dynatrace endpoint. 
+https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[The Dynatrace API token documentation] contains more information on how to create an API token.
+The 'Ingest metrics' (`metrics.ingest`) permission is required on the token in order to ingest metrics.
+More information on how to use the v2 API is provided <<bookmark-apiv2, below>>.
+
 [source,java]
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String uri() {
+        // Read the URI from the environment variables or a config file
+        // Should contain the path, e. g.: https://your-environment-id.live.dynatrace.com/api/v2/metrics/ingest
+        return MY_DYNATRACE_URI;
```

Comment:
```suggestion
        return MY_DYNATRACE_URI; // or `return "";` to automatically use the local OneAgent endpoint
```
Returning null does not work here, right?

Reply:
iirc all methods except `get` are `@NonNull`